packages/cli/src/lib/tunneling/index.ts

```diff
@@ -111,3 +111,19 @@ export const ensureTunnelKeyPair = async (
   await store.setTunnelingKey(Buffer.from(keyPair.privateKey))
   return keyPair
 }
+
+interface Props {
```

Comment:
nit: prefer types over interfaces unless needed (e.g, recursive types, declaration merging). also - if it's only used once, define it inline.